sim/druid/enrage.go

```diff
@@ -0,0 +1,65 @@
+package druid
+
+import (
+	"time"
+
+	"github.com/wowsims/wotlk/sim/core"
+)
+
+func (druid *Druid) registerEnrageSpell() {
+	actionID := core.ActionID{SpellID: 5229}
+	rageMetrics := druid.NewRageMetrics(actionID)
+
+	instantRage := 20.0
+	if druid.Talents.Intensity == 1 {

```

Comment:
This can be written more cleanly as `instantRage := []float64{20, 24, 27, 30}[druid.Talents.Intensity]`